test/fixtures/schemas/io/confluent/MailgunEvent.avsc

```diff
@@ -0,0 +1,174 @@
+{
+  "namespace": "io.confluent",
+  "name": "MailgunEvent",
+  "type": "record",
+  "fields": [
+    { "name": "event", "type": "string" },
```

Comment:
I think this is not that critical suggestion, but can I ask you to format it like all other schemas?

Reply:
I reformatted it again (earlier one was formatted using prettier) though not sure if it's following the exact format used in other files.